analysis/plot.py

```diff
@@ -0,0 +1,10 @@
+import matplotlib.pyplot as plt
+import numpy as np
+  
+data = np.genfromtxt('analysis/analysis.csv', delimiter=',', names=['type', 'dt', 'N'])
+
+plt.plot(data['N'], data['dt'], color='r', label='the data')
```

Comment:
isnt this already doing whats required ?

Reply:
yeah idk what im doing in python 😂

Reply:
just make it look good like this one

![plot](https://user-images.githubusercontent.com/55190601/119512323-d451c100-bd90-11eb-875e-3d74a5bb7124.png)